test/e2e/main/main_test.go

```diff
@@ -83,6 +84,7 @@ func TestUpdateOperatorPassword(t *testing.T) {
 	spec.Name = name
 	// Register it
 	testKube.CreateInfinispan(spec, tutils.Namespace)
+	defer testKube.CleanNamespaceAndLogOnPanic(tutils.Namespace)
```

Comment:
@durgeshanaokar it's required to replace DeleteInfinispan with this function
Also need to validate, is this work with the Batch entries to be removed and traced